connector/google/google.go

```diff
@@ -254,14 +259,23 @@ func (c *googleConnector) getGroups(email string) ([]string, error) {
 		for _, group := range groupsList.Groups {
 			// TODO (joelspeed): Make desired group key configurable
 			userGroups = append(userGroups, group.Email)
+
+			if groupsIndirectFetch {
+				indirectGroups, err := c.getGroups(group.Email, groupsIndirectFetch)
```

Comment:
Quoting from https://developers.google.com/admin-sdk/directory/v1/guides/manage-groups#get_all_member_groups:

>The userKey can be the user's primary email address, the user's alias email address, **a group's primary email address**, a group's email alias, or the user's unique id

And when a group is provided as userKey the parent groups will be returned.

Reply:
Would be good to add some of this context into the function as a comment IMO